config/production.ini

```diff
@@ -42,10 +42,6 @@ beaker.session.url=%(sqlalchemy.url)s
 beaker.session.timeout=3000
 beaker.session.lock_dir=/var/tmp/ckan/lock
 
-# `paster make-config` generates a unique value for this each time it generates
-# a config file.
-app_instance_uuid = {{ckan_instance_uuid}}
```

Comment:
Why remove this?

Reply:
This is no longer utilized in CKAN in any way. We had as part of the ticket to add it as a secret, but that becomes something we need to manage that we no longer need. See comment in thread starting [here](https://github.com/GSA/datagov-deploy/issues/2603#issuecomment-778002836)

Reply:
I still see the code in paster. Why not just set this to a guid? What if ckan does care about this value in the future or paster uses it for something that we didn't see?

Reply:
And the docs don't mention it as a secret so I would not treat it as such.